stl/inc/format

```diff
@@ -191,6 +213,32 @@ constexpr const _CharT* _Parse_width(const _CharT* _Begin, const _CharT* _End, _
     return _Begin;
 }
 
+template <class _CharT, _Parse_spec_callbacks<_CharT> _Callbacks_type>
+constexpr const _CharT* _Parse_precision(const _CharT* _Begin, const _CharT* _End, _Callbacks_type&& _Callbacks) {
+    ++_Begin;

```

Comment:
@CaseyCarter noted that this doesn't handle empty ranges - is that a concern? (I know it's internal machinery so I don't know the entire context.) Should there at least be a precondition comment?

In general, we don't bother documenting Standard function interfaces very much (that is what the Standard is for) but for internal machinery with non-obvious preconditions, it is very valuable to add such comments.

Reply:
yeah it doesn't matter, it's only called once you see a '.'